src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/turtles/TurtleChunkyUpgrade.java

```diff
@@ -34,10 +35,16 @@ protected ChunkyPeripheral buildPeripheral(@NotNull ITurtleAccess turtle, @NotNu
 
     @Override
     public void update(@NotNull ITurtleAccess turtle, @NotNull TurtleSide side) {
-        //Add a chunk to the Chunk Manager every 10 ticks, if it's not already forced.
-        //The turtle can move, so we need to do that.
+        // Add a chunk to the Chunk Manager every 10 ticks, if it's not already forced.
+        // The turtle can move, so we need to do that.
         super.update(turtle, side);
         if (APConfig.PERIPHERALS_CONFIG.enableChunkyTurtle.get()) {
+            updateTick++;
```

Comment:
This currently does not seem to do anything
Shouldn't it be removed or commented out